style.css

```diff
@@ -0,0 +1,60 @@
+body {
+    font-family: Arial, Helvetica, sans-serif;
+}
+.container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding-top: 50px;
+}
+.app {
+    width: 60%;
+}
+@media (max-width:800px){
+    .app {
+        width: 90%;
+    }
+}
```

Comment:
Loved that you have used media queries to build a responsive app. but make sure to take a mobile first approach to build web UIs